kview/src/main/java/dev/ikm/komet/kview/mvvm/view/journal/JournalController.java

```diff
@@ -408,6 +414,15 @@ public void initialize() {
         });
 
         workspace.addEventHandler(MouseEvent.MOUSE_CLICKED, this::onMouseClickedOnDesktopSurfacePane);
+        refreshCalculatorEventSubscriber = evt -> {
+            LOG.info("Refresh Calculator Event");
+            if(evt.getEventType() == GLOBAL_REFRESH) {
+                LOG.info("Global Refresh EventYpe...");
```

Comment:
update the log message to output the `GLOBAL_REFRESH`. tip: Debugging event messages it's often a good idea to show the name of the event types and topics in the console logs.